plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -109,31 +106,23 @@ Config setup() {
     }
 
     /**
-     * Outputs information about plugin txt file selected from CLI Option
+     * Outputs information about plugin txt or yaml file selected from CLI Option. Throws a PluginInputException if the
+     * file does not exist.
      *
-     * @return plugin txt file passed in through CLI, or null if user did not pass in txt file
+     * @return plugin txt or yaml file passed in through CLI, or null if user did not pass in a txt file
      */
-    private File getPluginTxt() {
-        if (pluginTxt == null) {
-            System.out.println("No .txt file containing list of plugins to be downloaded entered.");
+    private File getPluginFile() {
+        if (pluginFile == null) {
+            System.out.println("No .txt or .yaml file containing list of plugins to be downloaded entered.");
         } else {
-            System.out.println("File containing list of plugins to be downloaded: " + pluginTxt);
-        }
-        return pluginTxt;
-    }
-
-    /**
-     * Outputs information about plugin yaml file option selected from CLI Option
-     *
-     * @return plugin yaml file passed in through CLI, or null if user did not pass in a yaml file
-     */
-    private File getPluginYaml() {
-        if (pluginYaml == null) {
-            System.out.println("No .yaml file containing list of plugins to be downloaded entered.");
-        } else {
-            System.out.println("Yaml file containing list of plugins to be downloaded: " + pluginYaml);
+            if (Files.exists(pluginFile.toPath())) {
+                System.out.println("File containing list of plugins to be downloaded: " + pluginFile);
+            }
+            else {
+                throw new PluginInputException("File containing list of plugins does not exist");
```

Comment:
Include the file path please,
```suggestion
                throw new PluginInputException("File containing list of plugins does not exist " + pluginFile.toPath());
```